docs/adding-new-platforms.md

```diff
@@ -0,0 +1,81 @@
+## Adding new platforms
+
+Depending on what type of platform you're adding, the instructions here will
+vary.
+
+### Adding a brand new platform
+
+- Create a new file for your mainboard in `platform-configs/`. For most
+  platforms this file will be called `[platform-vendor]-[platform-model].robot`.
+- Add the following at the top of your platform config:
+
+```robot
+*** Settings ***
+Resource    default.robot
+```
+
+- Copy the contents of `include/default.robot` to your platform config
+- Modify the file for your platform:
+    - Modify the settings appropriately for your mainboard
+    - Remove any unmodified lines - they will be sourced from `default.robot`
+- Add the platform configuration to `variables.robot:
+    - Create a new configuration of RTE, if you are using one, e.g.:
+
+    ```robot
+    &{RTE11}=                   ip=192.168.10.174    cpuid=02c000426621f7ea    pcb_rev=1.0.0
```

Comment:
Can w try to remove unused values and at least not suggest to add them in documentation for new platforms? I image following are not used (need to be checked)
- cpuid
- pcb rev
- board-revision (?)
- env
- fw type

Reply:
Started work on removing unused variables: daa27b79c5ca929fd66963717b54205d85f519ef f6cfcd1883c2c8f023149062acb5f43af98d2daf - it makes more sense to keep them in snipeit than here.

Reply:
f72a7d83bbd9d7b5a4eb9bf3a2ce6ffb14bad71f

Reply:
I see that cpuid is used in some keywords, not sure if *these* are actually used somewhere, leaving it as-is for now.